.ci/create-release-github.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "${GH_WRITE_TOKEN}" | gh auth login --with-token
+
+BASE_BUILD_IMAGE=${BASE_BUILD_IMAGE:-"opentelemetry/opentelemetry-operator"}
+OPERATOR_VERSION=${OPERATOR_VERSION:-$(git describe --tags)}
+if [[ $OPERATOR_VERSION == v* ]]; then
+    OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d\.]+)")
+    MAJOR_MINOR=$(echo ${OPERATOR_VERSION} | awk -F. '{print $1"."$2}')
```

Comment:
Where is this used?

Reply:
I cleaned up this file a bit, so that most of it isn't required anymore.